### DISCUSSION TITLE
Disable or hide certain checkboxes in TreeView

### DISCUSSION THREAD

**Discussion by ykoitzsch** [vmware-clarity/ng-clarity#678] [Ideas]
I would like to have the possibility to use a Checkbox-Treeview and be able to configure specific nodes with a disabled or hidden property so their checkboxes won't be shown or can't be actively selected.

When one node has the selected property, all other nodes in the tree receive a checkbox.

**Comment by kevinbuhmann** [comment]
There is not built-in option to show a checkbox for a subset if the tree nodes. You can choose not to use the built-in in tree nodes selection feature and conditionally project your own checkbox in the tree node content.
